model-engine/model_engine_server/api/model_endpoints_v1.py

```diff
@@ -92,13 +92,6 @@ async def create_model_endpoint(
             status_code=404,
             detail="The specified model bundle could not be found.",
         ) from exc
-    except Exception as exc:
```

Comment:
removing generic exception handling here since it will be caught by the exception handling middleware.